index.bs

```diff
@@ -2739,6 +2881,39 @@ an optional [=boolean=] |isNoised| (default false):
 1. Run [=obtain and deliver a verbose debug report=] with « |data| », |source|'s [=attribution source/reporting origin=],
     and |source|'s [=attribution source/fenced=].
 
+To <dfn>obtain and deliver an aggregatable debug report on source registration</dfn>
+given a [=source debug data type=] |dataType| and an [=attribution source=] |source|:
+
+1. If |source|'s [=attribution source/fenced=] is true, return.
+1. Let |config| be |source|'s [=attribution source/aggregatable debug reporting config=].
+1. Let |debugDataMap| be |config|'s [=aggregatable debug reporting config/debug data=].
+1. If |debugDataMap| [=map/is empty=], return.
+1. Let |contributions| be a new [=list=].
+1. Let |dataTypeToReport| be |dataType|.
+1. If |dataTypeToReport| is "<code>[=source debug data type/source-success=]</code>"
+    and |isNoised| is true, set |dataTypeToReport| to "<code>[=source debug data type/source-noised=]</code>".
```

Comment:
The write here is unused. Maybe you meant to change the value of `|dataType|` itself?

Reply:
Updated, thanks.